rllib/examples/offline_rl.py

```diff
@@ -38,7 +38,7 @@
     config["bc_iters"] = 0
     config["clip_actions"] = False
     config["normalize_actions"] = True
-    config["replay_buffer_config"]["learning_starts"] = 256
```

Comment:
CQL does not need RB.

Reply:
@kouroshHakha The complete example is actually broken.
See https://github.com/ray-project/ray/issues/26665.
Not fixing this in this PR though.

Reply:
Adding to this so that it will not block this PR: The example is not part of CI and so was forgotten when we changed CQL.

Reply:
it shouldn't be broken anymore ?? see https://github.com/ray-project/ray/pull/27451